LibreNMS/Authentication/ActiveDirectoryAuthorizer.php

```diff
@@ -168,6 +169,28 @@ public function getUserid($username)
         return -1;
     }
 
+    public function getUsersAMAccountName($username)
+    {
+        $connection = $this->getConnection();
```

Comment:
This is run before bind, so it will fail unless an ad_bind account is set.

Reply:
I have activated Anonymous bind on my test AD domain.
Commented in config.php :
```
#$config['auth_ad_binduser'] = 'ldap_librenms';          // bind user (non-admin)
#$config['auth_ad_bindpassword'] = 'mypass'; 
```

So far so good no problem to authenticate with either the dgsd or dogus@lab.local.

But I have this error : 
```
./scripts/auth_test.php -u dgsd
Authentication Method: active_directory
Success
Could not anonymous bind to AD
Could not bind to AD, you will not be able to use the API or alert AD users
```
I have the same error "Could not bind to AD, you will not be able to use the API or alert AD users" even with auth_ad_binduser and auth_ad_bindpassword set, and current librenms master.